indico/modules/categories/blueprint.py

```diff
@@ -39,6 +40,9 @@
 _bp.add_url_rule('/manage/move', 'move-contents', RHCategoryMoveContents)
 _bp.add_url_rule('/manage/sort-subcategories', 'sort_subcategories', RHSortSubcategories, methods=('POST',))
 _bp.add_url_rule('/manage/create-subcategory', 'create_subcategory', RHCreateCategory, methods=('GET', 'POST'))
+_bp.add_url_rule('/manage/delete-events', 'delete_events', RHDeleteEvents, methods=('GET', 'POST'))
```

Comment:
`/manage/events/move`, `/manage/events/delete`? that would fit `/manage/subcategories/...` (introduced in #2478)